### PR TITLE
Add Panel

### PR DESCRIPTION
**Pull request by VitaliKozhurov**

### CODE REVIEW

File: src/components/ui/panel/Panel.module.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.root {
+  display: flex;
```
Comment (DenisKulik) [inline src/components/ui/panel/Panel.module.scss:2]: use flex mixin

File: src/components/ui/panel/Panel.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  onChangeTabValue: (value: string) => void
+  sliderValue: number[]
+  minSliderValue: number
+  maxSliderValue: number
+  sliderLabel: string
+  onChangeSliderValue: (value: number[]) => void
+  onClearFilter: () => void
+}
+
+export const Panel = (props: PanelProps): JSX.Element => {
```
Comment (vitalyabedik) [inline src/components/ui/panel/Panel.tsx:21]: destruct props from ( )

File: src/components/ui/panel/Panel.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    onChangeTabValue,
+    onChangeSliderValue,
+    onClearFilter,
+  } = props
+
+  return (
+    <div className={s.root}>
+      <Input className={s.input} value={inputValue} onChangeValue={onChangeInputValue} />
+      <Tabs className={s.tabs} label={tabLabel} value={tabValue} onValueChange={onChangeTabValue}>
+        <TabItem value={'myCards'}>My Cards</TabItem>
```
Comment (vitalyabedik) [inline src/components/ui/panel/Panel.tsx:40]: use "" instead {""}

File: src/components/ui/tabs/Tabs.module.scss [hunk 4]
```diff
@@ -1,75 +1,8 @@
-$border: 1px solid var(--color-dark-300);
-$border-active: 1px solid var(--color-accent-500);
-$border-focus: 1px solid var(--color-info-900);
-
 .root {
   width: 100%;
 
   .list {
     display: flex;
```
Comment (vitalyabedik) [inline src/components/ui/tabs/Tabs.module.scss:5]: use flex mixin

File: src/components/ui/tabs/Tabs.stories.tsx [hunk 5]
```diff
@@ -4,6 +4,8 @@ import type { Meta, StoryObj } from '@storybook/react'
 
 import { Tabs, TabsProps } from './Tabs'
 
+import { TabItem } from '@components/ui/tabs/tabItem'
```
Comment (vitalyabedik) [inline src/components/ui/tabs/Tabs.stories.tsx:7]: use relative import

File: src/components/ui/tabs/Tabs.stories.tsx [hunk 6]
```diff
@@ -13,35 +15,51 @@ const meta: Meta<typeof Tabs> = {
...
+  )
 }
 
 export const Default: Story = {
   args: {
     value: '1',
-    tabs: initialTabsState,
+    children: (
+      <>
+        <TabItem value={'1'}>{'Switcher_1'}</TabItem>
```
Comment (vitalyabedik) [inline src/components/ui/tabs/Tabs.stories.tsx:37]: {''} -> ""

File: src/components/ui/tabs/Tabs.stories.tsx [hunk 7]
```diff
@@ -13,35 +15,51 @@ const meta: Meta<typeof Tabs> = {
...
+        <TabItem value={'2'}>{'Switcher_2'}</TabItem>
```
Comment (vitalyabedik) [inline src/components/ui/tabs/Tabs.stories.tsx:38]: the same

File: src/components/ui/panel/Panel.module.scss (hunk 1 above)
Comment (VitaliKozhurov) [reply src/components/ui/panel/Panel.module.scss:2]: Add mixin for flex

File: src/components/ui/panel/Panel.tsx (hunk 3 above)
Comment (VitaliKozhurov) [reply src/components/ui/panel/Panel.tsx:40]: Fix)

File: src/components/ui/panel/Panel.tsx (hunk 2 above)
Comment (VitaliKozhurov) [reply src/components/ui/panel/Panel.tsx:21]: Done!

File: src/components/ui/panel/Panel.tsx (hunk 3 above)
Comment (VitaliKozhurov) [reply src/components/ui/panel/Panel.tsx:40]: Add mixin for flex

File: src/components/ui/tabs/Tabs.module.scss (hunk 4 above)
Comment (VitaliKozhurov) [reply src/components/ui/tabs/Tabs.module.scss:5]: Add mixin for flex

File: src/components/ui/tabs/Tabs.stories.tsx (hunk 5 above)
Comment (VitaliKozhurov) [reply src/components/ui/tabs/Tabs.stories.tsx:7]: Done!

File: src/components/ui/tabs/Tabs.stories.tsx (hunk 6 above)
Comment (VitaliKozhurov) [reply src/components/ui/tabs/Tabs.stories.tsx:37]: Done!

File: src/components/ui/tabs/Tabs.stories.tsx (hunk 7 above)
Comment (VitaliKozhurov) [reply src/components/ui/tabs/Tabs.stories.tsx:38]: Done!

**Review by vitalyabedik — APPROVED** [review]
Good!

**Pull request merged**